en/guide/vtol.md

```diff
@@ -0,0 +1,40 @@
+# VTOL Support
+
+DroneCore has basic support for VTOL vehicles. 
+
+> **Note** Most design and test effort has gone into multicopter support. 
+> [Get in touch](../README.md#getting-help) if you would like to help enhance the VTOL/Fixed-wing experience! 
+
+## Supported Functionality

```

Comment:
@julianoes  Can you please re-review this section - which I updated to make it clear that MC takeoff in missions then manual transition to FW is supported, but VTOL takeoff in missions with automatic transition to FW is not.

Reply:
Yep all good.